lib/screens/login_screen.dart

```diff
@@ -86,103 +86,119 @@ class _LoginScreenState extends State<LoginScreen>
     super.dispose();
   }
 
+  void _setSystemNavigationBarColor(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(
+      SystemUiOverlayStyle(
+        statusBarColor: Theme.of(context).canvasColor,
+        statusBarIconBrightness: Brightness.dark,
+      ),
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    return PicosScreenFrame(
-      body: Center(
-        child: PicosBody(
-          child: Column(
-            children: <Widget>[
-              const Image(
-                image: AssetImage('assets/PICOS_Logo_RGB.png'),
-              ),
-              RichText(
-                text: TextSpan(
-                  style: const TextStyle(
-                    height: 2,
-                    color: Colors.black,
-                  ),
-                  children: <TextSpan>[
-                    TextSpan(
-                      text:
-                          '${AppLocalizations.of(context)!.welcomeToPICOS},\n',
-                      style: const TextStyle(
-                        fontWeight: FontWeight.bold,
-                      ),
-                    ),
-                    TextSpan(
-                      text: AppLocalizations.of(context)!
-                          .thankYouForParticipation,
+    _setSystemNavigationBarColor(context);
+    return SafeArea(
+      child: Scaffold(
+        appBar: PreferredSize(
+          preferredSize: const Size.fromHeight(0.0),
+          child: AppBar(),
+        ),
+        body: Center(
+          child: PicosBody(
+            child: Column(
+              children: <Widget>[
+                const Image(
+                  image: AssetImage('assets/PICOS_Logo_RGB.png'),
+                ),
+                RichText(
+                  text: TextSpan(
+                    style: const TextStyle(
+                      height: 2,
+                      color: Colors.black,
                     ),
-                  ],
+                    children: <TextSpan>[
+                      TextSpan(
+                        text:
+                            '${AppLocalizations.of(context)!.welcomeToPICOS},\n',
```

Comment:
Try not to make lines above 80 characters.

Reply:
Resolved in e9c442a7f063ca9fcd15e451ed0870d25ed632c2